client/blocks/get-apps/apps-badge.scss

```diff
@@ -0,0 +1,20 @@
+.get-apps__app-badge {
+	display: inline-block;
+	img {
+		max-height: 41px;
+		margin: 9px 0;
+		width: auto;
+	}
+
+	&.android-app-badge {
+		img {
+			max-height: 59px;
+			margin-top: 0;
+			margin-bottom: 0;
+			margin-right: 0;
+		}
+		& + .ios-app-badge img {
+			margin-left: 0;
```

Comment:
These styles and the `+` selector look like unneeded complexity. Maybe it was necessary once, but isn't now. Just setting the right dimensions and margins for `.android-app-badge` and `.ios-app-badge` should be enough.